pkgs/os-specific/linux/hwdata/default.nix

```diff
@@ -19,7 +19,7 @@ stdenv.mkDerivation rec {
 
   outputHashMode = "recursive";
   outputHashAlgo = "sha256";
```

Comment:
Why is this a fixed output derivation? I can't look it up because the PR that introduced it is no longer available.

Reply:
Good question. I don't have the slightest idea. It's a pity the referenced pull request #44606 is no longer available.